### DISCUSSION TITLE
Feature Request: News Pages

### DISCUSSION THREAD

**Discussion by sloanxue** [zencart/zencart#6275] [Ideas]
Zencart is a powerful online store system that is very well-established in shopping features.
However, compared to WooCommerce(Word Press), and Shopfiy, there is a lack of a news system.
I used to use the News Box Manger plugin, but it doesn't work in the new version.
Whether it's from the perspective of website functionality or SEO, the news system i.
I'm really looking forward to adding news or blog features to the new ZenCart.

**Comment by lat9** [comment]
Have you tried the updated [News Box Manager - V3](https://www.zen-cart.com/showthread.php?226052-News-Box-Manager-v3-0-0-Support-Thread)?  I'm using that on my website that's running zc200-beta and PHP 8.3.

**Comment by scottcwilson** [comment]
Excellent suggestion.

More ideas on adding content to your site:
https://docs.zen-cart.com/user/customizing/add_pages/

**Reply by sloanxue** [reply]
Thank you

**Comment by sloanxue** [comment]
> Have you tried the updated [News Box Manager - V3](https://www.zen-cart.com/showthread.php?226052-News-Box-Manager-v3-0-0-Support-Thread)? I'm using that on my website that's running zc200-beta and PHP 8.3.

Thank you, I am going to try V3.
I think the plugin will be complicated in case the store owener use Usu, sitemapXML and json LD plugin.
So I wish the new zencart can integrate news page feature, it is better to have commet feature.
**I also hope new zencart could integrate some important plugin such as USU, sitemap XML, Json LD, Image handler, Quick update**
